pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1853,6 +1854,7 @@ func TestPriorityQueue_AssignedPodAdded(t *testing.T) {
 	defer cancel()
 
 	affinityPod := st.MakePod().Name("afp").Namespace("ns1").UID("afp").Annotation("annot2", "val2").Priority(mediumPriority).NominatedNodeName("node1").PodAffinityExists("service", "region", st.PodAffinityWithRequiredReq).Obj()
+	spreadPod := st.MakePod().Name("tsp").Namespace("ns1").UID("tsp").SpreadConstraint(1, "node", v1.DoNotSchedule, nil, nil, nil, nil, nil).Obj()
```

Comment:
Please instead create a new test and rename the existing one to be specific about pod affinity (the comment already talks about pod affinity specifically).

Or maybe it can be converted into a test table.

Reply:
I converted into a table driven one.